src/bci_build/package/php/README.md.j2

```diff
@@ -1,4 +1,4 @@
-# The PHP {{ image.version }} container image
+# The PHP {{ image.tag_version }} container image
```

Comment:
This is now a regex in the rendered version

Reply:
no, that was php-fpm/README.md.j2 missing a fix. fixed now also.